apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -226,5 +235,39 @@ internal string GetTraceName()
         /// </summary>
         internal IEnumerable<ILogEntryLabelProvider> GetLabelProviders() =>
             _serviceProvider?.GetService<IEnumerable<ILogEntryLabelProvider>>();
+
+        /// <summary>
+        /// For diagnostic purposes. Builds and returns the URL where the entries logged by
+        /// this <see cref="GoogleLogger"/> can be seen on the GCP Stackdriver Logging Console.
+        /// </summary>
+        public Uri GetGcpConsoleLogsUrl()
+        {
+            IList<string> parameters = new List<string>
+            {
+                $"resource={_loggerOptions.MonitoredResource.Type}",
+                $"minLogLevel={(int)_loggerOptions.LogLevel.ToLogSeverity()}",
+                $"logName={_fullLogName}",
+                _logTarget.Kind == LogTargetKind.Project ? $"project={_logTarget.ProjectId}" :
```

Comment:
I think this would be easier to format outside the collection initializer:

```text
string target =
    _logTarget.Kind == LogTargetKind.Project ? $"project={_logTarget.ProjectId}" :
    _logTarget.Kind == LogTargetKind.Organization ? $"organizationId={_logTarget.OrganizationId}" :
    throw new InvalidOperationException($"Unrecognized LogTargetKind: {_logTarget.Kind}");
```

Then use `target` in `parameters`